FrontEnd/scripts/works.js

```diff
@@ -50,31 +56,73 @@ async function displayWork(worksCategory) {
     workImage.setAttribute("alt", work.title);
     workFigCaption.textContent = work.title;
 
-    // NOTE: adds to elements to the parent html container "gallery" cold before
+    // NOTE: adds to elements to the parent html container called before
     workFigure.appendChild(workImage);
     workFigure.appendChild(workFigCaption);
-    worksContainer.appendChild(workFigure);
+    target.appendChild(workFigure);
   });
 }
 
+// NOTE: Ici commence l'execution du code
+//
+//
+if (sessionStorage.getItem("token") != null) {
+  editionModeBand.style.display = "flex";
+  for (editionButton of editionModeButtons) {
+    editionButton.style.display = "flex";
+  }
+}
+
+// NOTE: opens the modal
+modifyBtn.addEventListener("click", () => {
```

Comment:
```suggestion
modifyBtn.addEventListener("click", function openModal () {
```

You could replicate this for other callbacks: **name the function**; it will help to debug.